src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -289,6 +308,9 @@ export class CalciteAlert {
 
   /** close and emit the closed alert and the queue */
   private closeAlert = (): void => {
+    if (this.autoDismissTimeout) {
+      this.autoDismissTimeout = null;
```

Comment:
Unless I'm missing something, you could always clear the timeout here without checking.

Reply:
nice catch 🏈